app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java

```diff
@@ -50,6 +54,8 @@
     private static final String ARGUMENT_HEIGHT = "height";
     private static final String ARGUMENT_OVERLAY = "translucent";
 
+    private String pasteString = "";
```

Comment:
Should this be a global variable? Its value is initialized when view is created, but the paste can feasibly change since then.